labs/Private-transaction-families.md

```diff
@@ -0,0 +1,48 @@
+# Lab Name
+
+Private-transaction-families
+
+# Short Description
+
+Private-transaction-families system provides a mechanism for privacy over Hyperledger Sawtooth by enforcing a policy of access control to the ledger and sensitive state content based on the client’s role trying to access the data.
+
+The system is based on a generic distributed ledger infrastructure (Hyperledger Sawtooth) and an Intel® SGX (Software Guard Extensions) ‘hardened’ Transaction Processor which provides a mechanism to enable Hyperledger Sawtooth to contain private information that is both not publicly available but required to validate transactions. This solution supports encryption of information in transactions and blocks while allowing the ledger to validate the information in those transactions in all the nodes and allow them to reach a consensus on the current state of the ledger.
+
+Private-transaction-families provides benefits for both application developers seeking to define and implement privacy-preserving distributed ledgers, and for service providers seeking to provide blockchain services with access restriction to the ledger data.
+
+# Scope of Lab
+
+Private-transaction-families system provides a mechanism for privacy over Hyperledger Sawtooth by enforcing a policy of access control to the ledger and sensitive state content based on the client’s role trying to access the data.
+
+The system is based on a generic distributed ledger infrastructure (Hyperledger Sawtooth) and an Intel® SGX (Software Guard Extensions) ‘hardened’ Transaction Processor which provides a mechanism to enable Hyperledger Sawtooth to contain private information that is both not publicly available but required to validate transactions. This solution supports encryption of information in transactions and blocks while allowing the ledger to validate the information in those transactions in all the nodes and allow them to reach a consensus on the current state of the ledger.
+
+# Documentation
```

Comment:
Remove, as this is not part of the template and does not currently contain any information.